src/V3SchedReplicate.cpp

```diff
@@ -124,6 +124,8 @@ class VarVertex final : public Vertex {
         if (varp()->isPrimaryInish() || varp()->isSigUserRWPublic() || varp()->isWrittenByDpi()) {
             addDrivingRegions(INPUT);
         }
+        // All suspendable and downstream from suspendable logic should be active
```

Comment:
```suggestion
Combinational logic driven from a suspendable processes must be present in the 'act' region
```

Note: I consistently use `'act' region` and `'nba' region` (and 'ico' and 'stl) to refer to the Verilator created specific functions, rather than 'active' or 'NBA' which I use in the IEEE standard sense. There is a subtle but important difference (namely some IEEE 'active' execute in the Verilator 'nba' region).

Reply:
The region flag is `ACTIVE` though?
```
ACTIVE = 0x2,  // Variable/logic is driven from 'act' region logic
```

Reply:
Looks like it, sorry for the inconsistency there